crates/release_plz_core/src/git/backend.rs

```diff
@@ -147,6 +160,7 @@ pub struct GitLabMr {
     pub source_branch: String,
     pub title: String,
     pub description: String,
+    pub labels: Vec<Label>,
```

Comment:
are you sure this is how the gitlab merge request response looks like?
[Here](https://docs.gitlab.com/ee/api/merge_requests.html#get-single-mr) it looks like it's just a vector of strings.

If you confirm, can you fix this?

Reply:
I am pushing a fix for this now